### ISSUE TITLE
In package.json, older of version of ldap-filter is used.

### ISSUE THREAD

**Issue by farheen14aslam** [joyent/node-ldapjs#467]
In package.json, the older version of ldap-filter is used, i.e 0.2.2, which has the dependency on assert-plus older version(0.1.5). The older version of assert-plus is having the errors when strict() is used. This issue is resolved in the newer version of assert-plus. So if we update the "ldap-filter": "0.2.2", to the newer version, the issue can be resolved.

**Comment by mkhanal** [comment]
Is there any update on this. There are a lot of forks floating around that have made changes just to make this work.
This also means any application using babel is not able to use the module which is a huge bummer.

**Comment by farheen14aslam** [comment]
Yes .. You are correct. This problem was mentioned before as well but no update on the same. The only way I could get it work is to fork the project and make the necessary changes as per my project requirement.

**Comment by jsumners** [comment]
⚠️ This issue has been locked due to age. If you have encountered a recent
problem that seems to be covered by this issue, please open a new issue.

Please include a [minimal reproducible example](https://en.wikipedia.org/wiki/Minimal_reproducible_example)
when opening a new issue.
